subsystem/core/src/main/java/org/wildfly/extension/camel/parser/CamelSubsystemAdd.java

```diff
@@ -101,9 +106,11 @@ public void execute(final DeploymentProcessorTarget processorTarget) {
                 processorTarget.addDeploymentProcessor(CamelExtension.SUBSYSTEM_NAME, Phase.PARSE, PARSE_CAMEL_CONTEXT_DESCRIPTORS, new CamelContextDescriptorsProcessor());
                 processorTarget.addDeploymentProcessor(CamelExtension.SUBSYSTEM_NAME, Phase.DEPENDENCIES, DEPENDENCIES_CAMEL_INTEGRATION, new CamelIntegrationProcessor());
                 processorTarget.addDeploymentProcessor(CamelExtension.SUBSYSTEM_NAME, Phase.DEPENDENCIES, DEPENDENCIES_CAMEL_WIRINGS, new CamelDependenciesProcessor());
+                processorTarget.addDeploymentProcessor(CamelExtension.SUBSYSTEM_NAME, Phase.DEPENDENCIES, DEPENDENCIES_CAMEL_ENDPOINT_DEPLOYMENT_SCHEDULER, new CamelEndpointDeploymentSchedulerProcessor());
```

Comment:
I saw the comment in `CamelEndpointDeploymentSchedulerService`:

_The split between `CamelEndpointDeploymentSchedulerService` & `CamelEndpointDeployerService` is necessary because the requests to deploy HTTP endpoints may come in phases before the `CamelEndpointDeployerService` is available_

Is this correct? A Camel Context will only ever be started during `Phase.INSTALL`. So I'd only ever expect requests to deploy HTTP endpoints to come from this phase.

Reply:
Yes, I am quite sure, but I'll re-check.